apps/functions-subscription/api/internal.yaml

```diff
@@ -179,3 +179,6 @@ components:
           $ref: '#/components/schemas/CreatedAt'
         updatedAt:
           $ref: '#/components/schemas/UpdatedAt'
+    SubscriptionEvent:
+      type: object
+      $ref: '#/components/schemas/Subscription'
```

Comment:
**note**: Add the `kind` attribute to define the type of event.

Reply:
**thought(non-blocking)**: Maybe here we can create a different object, with less properties.
For instance, I think the `updatedAt` should not be a part of the `SubscriptionEvent` object.

But, we can do this kind of refactor later when we will have more knowledge of what we need to share on the queue.